gap_windows.go

```diff
@@ -381,3 +382,8 @@ func (d Device) RequestConnectionParams(params ConnectionParams) error {
 	// BluetoothLEDevice.RequestPreferredConnectionParameters.
 	return nil
 }
+
+// SetRandomAddress sets the random address to be used for advertising.
+func (a *Adapter) SetRandomAddress(mac MAC) error {
+	return errors.ErrUnsupported
```

Comment:
TODO or not possible to implement?

Reply:
I think TODO? @jagobagascon is the expert :smile:

Reply:
Then could be better to return `errors.ErrNotImplemented` (if we have one?)

Reply:
I don't think this can be set on Windows. I did not see any way of doing it in the [WinRT API](https://learn.microsoft.com/en-us/uwp/api/windows.devices.bluetooth.advertisement.bluetoothleadvertisementpublisher?view=winrt-26100)

Reply:
That was how it looked to me as well.